### DISCUSSION TITLE
[bug]:

### DISCUSSION THREAD

**Discussion by NicolasDorier** [lightningnetwork/lnd#8436] [Q&A] [answered]
### Background

I have several question with the below forced closed channel.
1. Why is it still in `pending_force_closing_channels` when the closing transaction `5a270e17d38653b30bf6a0f60a8045a5dc93df15084d72f3a10e4028209460de` has been confirmed for ages.
2. Why `limbo_balance` is 2244 when local_balance is `2994185` 

```json
    "pending_force_closing_channels":  [
        {
            "channel":  {
                "remote_node_pub":  "026c79920c8bdf86fd464c01e106ad84287ff6ee0d41d65e900d67e3ba91327f8a",
                "channel_point":  "b543e4454d03ddc60313ffc4f5c8db7aadf307686c1ec11a4e0cf0497c946add:1",
                "capacity":  "3000000",
                "local_balance":  "2994185",
                "remote_balance":  "0",
                "local_chan_reserve_sat":  "0",
                "remote_chan_reserve_sat":  "0",
                "initiator":  "INITIATOR_LOCAL",
                "commitment_type":  "ANCHORS",
                "num_forwarding_packages":  "0",
                "chan_status_flags":  "",
                "private":  false,
                "memo":  ""
            },
            "closing_txid":  "5a270e17d38653b30bf6a0f60a8045a5dc93df15084d72f3a10e4028209460de",
            "limbo_balance":  "2244",
            "maturity_height":  0,
            "blocks_til_maturity":  0,
            "recovered_balance":  "0",
            "pending_htlcs":  [
                {
                    "incoming":  false,
                    "amount":  "1914",
                    "outpoint":  "0b315cba26c3c69d2b1eec4a24e667c0b36312f4e983ebc64ab3c356b13e02d3:0",
                    "maturity_height":  816338,
                    "blocks_til_maturity":  -11246,
                    "stage":  1
                }
            ],
            "anchor":  "LIMBO"
        }
    ],
```

### Your environment

lnd v0.17.3 on docker

### Steps to reproduce

I don't know 

### Expected behaviour

1. The limbo balance should match the local balance.
2. The channel shouldn't be in the pending forced close channels anymore

### Actual behaviour

1. The limbo balance of a forced close channel has a different amount from the local balance.
2. The forced close channel has the closing transaction confirmed but is still in the list.

**Comment by bitromortac** [comment, marked as answer]
It looks like this is correct behavior, the limbo balance refers to the value of the anchor output (`"anchor": "LIMBO"`) plus the HTLC amount sitting in the first stage transaction (https://mempool.space/tx/999c6a9a7d86539f89894137a72374b6ce4f93c78e36040b8329fa03ab826ce5#vin=0). The HTLC output still sits there because it's uneconomical to spend. The to_local outpoint was already swept.

**Comment by NicolasDorier** [comment]
this make total sense thanks, need to fix our bug in btcpay then https://github.com/btcpayserver/BTCPayServer.Lightning/issues/156
